src/metakb/cli.py

```diff
@@ -7,401 +7,366 @@
-        help="Check normalizers database and load data if necessary.",
-    )
-    @click.option(
-        "--force_load_normalizers_db",
-        "-f",
-        is_flag=True,
-        default=False,
-        help=(
-            "Load all normalizers data into database. Overrides "
-            "--load_normalizers_db if both are selected."
-        ),
-    )
-    @click.option(
-        "--normalizers_db_url",
-        default="http://localhost:8000",
-        help=(
-            "URL endpoint of normalizers DynamoDB database. Set to "
-            "`http://localhost:8000` by default."
-        ),
-    )
-    @click.option(
-        "--load_latest_cdms",
-        "-l",
-        is_flag=True,
-        default=False,
-        help=(
-            "Clear MetaKB database and load most recent available source "
-            "CDM files. Does not run harvest and transform methods to "
-            "generate new CDM files. Exclusive with --load_target_cdm and "
-            "--load_latest_s3_cdms."
-        ),
-    )
-    @click.option(
-        "--load_target_cdm",
-        "-t",
-        type=click.Path(exists=True, dir_okay=False, readable=True, path_type=Path),
-        required=False,
-        help=(
-            "Load transformed CDM file at specified path. Exclusive with "
-            "--load_latest_cdms and --load_latest_s3_cdms."
-        ),
-    )
-    @click.option(
-        "--load_latest_s3_cdms",
-        "-s",
-        is_flag=True,
-        default=False,
-        required=False,
-        help=(
-            "Clear MetaKB database, retrieve most recent data available "
-            "from VICC S3 bucket, and load the database with retrieved "
-            "data. Exclusive with --load_latest_cdms and load_target_cdm."
-        ),
-    )
-    @click.option(
-        "--update_cached",
-        "-u",
-        is_flag=True,
-        default=False,
-        required=False,
-        help=(
-            "`True` if civicpy cache should be updated. Note this will take serveral"
-            "minutes. `False` if local cache should be used"
-        ),
-    )
-    async def update_metakb_db(
-        db_url: str,
-        db_username: str,
-        db_password: str,
-        load_normalizers_db: bool,
-        force_load_normalizers_db: bool,
-        normalizers_db_url: str,
-        load_latest_cdms: bool,
-        load_target_cdm: Optional[Path],
-        load_latest_s3_cdms: bool,
-        update_cached: bool,
-    ) -> None:
-        """Execute data harvest and transformation from resources and upload
-        to graph datastore.
-        """
-        if sum([load_latest_cdms, bool(load_target_cdm), load_latest_s3_cdms]) > 1:
-            CLI()._help_msg(
-                "Error: Can only use one of `--load_latest_cdms`, "
-                "`--load_target_cdm`, `--load_latest_s3_cdms`."
+    if not any([load_latest_cdms, load_target_cdm, load_latest_s3_cdms]):
+        if force_load_normalizers_db:
+            if normalizers_db_url:
```

Comment:
this is true by default since the default value for this param is `localhost:8000`. I think the only way to *not* override the DB_URL env vars would be to explicitly set it to `""` which seems a little weird. Might make more sense to not give `--normalizers_db_url` a default value, since the normalizers themselves already have default values in the `create_db()` methods.